--- a/chrome/browser/ui/accelerator_table.cc
+++ b/chrome/browser/ui/accelerator_table.cc
@@ -269,10 +269,8 @@ const AcceleratorMapping kDevToolsAccelerators[] = {
     {ui::VKEY_I, ui::EF_SHIFT_DOWN | ui::EF_CONTROL_DOWN, IDC_DEV_TOOLS},
     {ui::VKEY_J, ui::EF_SHIFT_DOWN | ui::EF_CONTROL_DOWN,
      IDC_DEV_TOOLS_CONSOLE},
     {ui::VKEY_E, ui::EF_SHIFT_DOWN | ui::EF_CONTROL_DOWN,
      IDC_DEV_TOOLS_INSPECT},
-    {ui::VKEY_C, ui::EF_SHIFT_DOWN | ui::EF_CONTROL_DOWN,
-     IDC_COPY_OR_INSPECT_SHORTCUT},
     {ui::VKEY_U, ui::EF_CONTROL_DOWN, IDC_VIEW_SOURCE},
 #endif  // !BUILDFLAG(IS_MAC)
 };
--- a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
+++ b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
@@ -938,16 +938,21 @@ TabDragController::Liveness TabDragController::ContinueDragging(
     if (current_drag_target_) {
       TabDragContext* delegate_context =
           current_drag_target_->OnTabDragUpdated(*this, point_in_screen);
       if (!delegate_context && current_state_ == DragState::kDraggingWindow) {
         delegate_context = attached_context_;
       }
       target_context = delegate_context;
     }
+#if BUILDFLAG(IS_LINUX)
+    if (!target_context) {
+      target_context = attached_context_;
+    }
+#endif
   }
 
   if (target_context != attached_context_) {
     is_dragging_new_browser_ = false;
     did_restore_window_ = false;
 
     return DragBrowserToNewTabStrip(target_context, point_in_screen);
   }
