--- a/chrome/browser/ui/accelerator_table.cc
+++ b/chrome/browser/ui/accelerator_table.cc
@@ -269,10 +269,8 @@ const AcceleratorMapping kDevToolsAccelerators[] = {
     {ui::VKEY_I, ui::EF_SHIFT_DOWN | ui::EF_CONTROL_DOWN, IDC_DEV_TOOLS},
     {ui::VKEY_J, ui::EF_SHIFT_DOWN | ui::EF_CONTROL_DOWN,
      IDC_DEV_TOOLS_CONSOLE},
     {ui::VKEY_E, ui::EF_SHIFT_DOWN | ui::EF_CONTROL_DOWN,
      IDC_DEV_TOOLS_INSPECT},
-    {ui::VKEY_C, ui::EF_SHIFT_DOWN | ui::EF_CONTROL_DOWN,
-     IDC_COPY_OR_INSPECT_SHORTCUT},
     {ui::VKEY_U, ui::EF_CONTROL_DOWN, IDC_VIEW_SOURCE},
 #endif  // !BUILDFLAG(IS_MAC)
 };
--- a/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
+++ b/chrome/browser/ui/views/tabs/dragging/tab_drag_controller.cc
@@ -1001,5 +1001,13 @@ TabDragController::Liveness TabDragController::DragBrowserToNewTabStrip(
   if (!target_context) {
+#if BUILDFLAG(IS_LINUX)
+    if (current_state_ == DragState::kDraggingTabs &&
+        attached_context_->GetPositioningDelegate()) {
+      dragging_tabs_session_->MoveAttached(point_in_screen);
+    }
+    return Liveness::kAlive;
+#else
     return DetachIntoNewBrowserAndRunMoveLoop(point_in_screen);
+#endif
   }

 #if defined(USE_AURA)
